This patch does not affect the behavior of the Makefile unless the added
variables LOFLAGS, ARFLAG, or LIBSOFLAGS are specified when the Makefile
is called.

reworked for libebml-1.3.0

--- libebml-1.3.0/make/linux/Makefile.orig	2013-03-19 19:42:58.000000000 +0100
+++ libebml-1.3.0/make/linux/Makefile	2015-05-17 15:46:54.901571939 +0200
@@ -64,6 +64,9 @@
 WARNINGFLAGS=-Wall -Wextra -Wno-unknown-pragmas -ansi -fno-gnu-keywords -Wshadow
 COMPILEFLAGS=$(WARNINGFLAGS) $(CXXFLAGS) $(CPPFLAGS) $(DEBUGFLAGS) $(INCLUDE)
 DEPENDFLAGS  = $(CXXFLAGS) $(INCLUDE)
+LOFLAGS=-fPIC
+ARFLAGS=rcvu
+LIBSOFLAGS=-Wl,-soname,
 
 ifeq (Darwin,$(shell uname -s))
 all: staticlib
@@ -84,14 +87,14 @@
 	$(CXX) -c $(COMPILEFLAGS) -o $@ $<
 
 %.lo: %$(EXTENSION)
-	$(CXX) -c $(COMPILEFLAGS) -fPIC -o $@ $<
+	$(CXX) -c $(COMPILEFLAGS) $(LOFLAGS) -o $@ $<
 
 $(LIBRARY): $(objects)
-	$(AR) rcvu $@ $(objects)
+	$(AR) $(ARFLAGS) $@ $(objects)
 	$(RANLIB) $@
 
 $(LIBRARY_SO): $(objects_so)
-	$(CXX) -shared -Wl,-soname,$(LIBRARY_SO_VER) -o $(LIBRARY_SO_VER) $(objects_so)
+	$(CXX) -shared $(LIBSOFLAGS)$(LIBRARY_SO_VER) -o $(LIBRARY_SO_VER) $(objects_so)
 	rm -f $(LIBRARY_SO)
 	ln -s $(LIBRARY_SO_VER) $(LIBRARY_SO)
 
